projects/dev/src/app/shared/constants/api.ts

```diff
@@ -5,3 +5,10 @@ export const GROUP_MEMBERS_API = {
 export const GROUP_REQUESTS_API = {
   sort: ["-at", "member_id"],
```

Comment:
Shouldn't "sort" be "defaultSort"? Do you really care about the default sort actually? If you do not provide the sort arg, the backend will apply its own default sort.

Reply:
It should be defaultSort.
But the frontend also needs to know what the default argument for sort is.
So I suggest using that constant.

Reply:
fixed.

Reply:
Related with the previous discussion, closing this one.